simpledotorg/simple-server
# Update reporting facility states to add diabetes outcomes

**Story card:** [sc-7793](https://app.shortcut.com/simpledotorg/story/7793/update-the-materialized-views-to-include-diabetes-outcomes)

## Because

ReportingFacilityStates doesn't include diabetes outcomes which are necessary to create DM reports

## This addresses

Add necessary diabetes outcome metrics to reporting facility states required to generate DM reports

## Test instructions

- Checkout to the `update-facility-states-to-collate-bs-data` branch
- Run migration `bundle exec rails db:migrate`
- Create a few patients and blood sugars for these patients
- Refresh the materialized view : `bundle exec rake db:refresh_reporting_views`
- You should be able to see appropriate data in the ReportingFacilityState table
